### DISCUSSION TITLE
docker-ce Update failed

### DISCUSSION THREAD

**Discussion by Ed1s0nZ** [Security-Onion-Solutions/securityonion#5022] [Unsupported Versions]
![image](https://user-images.githubusercontent.com/83812544/127798049-68c37792-0d6f-45dc-b690-99501805f966.png)
docker-ce 50 attempts to update failed
I tried to fix this with "apt-get update", but it didn't work

**Comment by TOoSmOotH** [comment]
Without more of the log I can only guess that it is unable to get the the Ubuntu Docker repo.
